### DISCUSSION TITLE
AC3 audio not an option

### DISCUSSION THREAD

**Discussion by martinbenny** [section83/MacYTDL#75] [General]
Him I download from SVTPLAY with YouTube-dl. It has the option to list all formats, on this page AC 6ch is available but neither yt-dlp or YouTube-dl pick this audio by default and I don't know how to select it in any way except manual for each file. Perhaps that is a function you could add? 

https://www.svtplay.se/video/e66EPAr/vem-ar-du-mamma-mu/avsnitt-6?info=visa

**Comment by section83** [comment]
Hello Martin, thank you for making this suggestion.  I will do something on this for the next release of MacYTDL – which I hope will be ready by the end of May.

MacYTDL does have a function which lists all the audio and video files available but, MacYTDL crashes on the data returned by svtplay.se. I will fix that in the next release.

I can't access the video you linked because it is geo-blocked. However, I did find a video which played [https://www.svtplay.se/video/KNnNdBr/detektiverna/nisse-ordnar-pusseldeckare?id=eYA7gvz]. The following audio files are available:

```
hls-AAC-190-2ch-Svenska             
hls-cmaf-full-AAC-190-2ch-Svenska   
hls-cmaf-full-AAC-30-2ch-Svenska    
hls-cmaf-full-AAC-60-2ch-Svenska    
hls-cmaf-lb-full-AAC-30-2ch-Svenska 
hls-cmaf-lb-full-AAC-60-2ch-Svenska 
hls-ts-avc-AAC-190-2ch-Svenska      
hls-ts-full-AAC-190-2ch-Svenska     
hls-ts-lb-full-AAC-190-2ch-Svenska  
dash-full-12                        mp4a.40.29  35k
dash-lb-full-6                      mp4a.40.29  35k
dash-full-11                        mp4a.40.5   67k
dash-lb-full-5                      mp4a.40.5   67k
dash-5                              mp4a.40.2  197k
dash-avc-5                          mp4a.40.2  197k
dash-full-10                        mp4a.40.2  197k
dash-hbbtv-avc-2                    mp4a.40.2  197k
dash-hbbtv-hevc-2                   mp4a.40.2  197k
```

Would one of those meet your need ?
